lms/templates/instructor/instructor_dashboard_2/discussions.underscore

```diff
@@ -1,5 +1,26 @@
 <!-- Discussion Topics. -->
 <div class="discussions-nav" id="discussions-management" tabindex="-1">
-    <div class="course-wide-discussions-nav"></div>
-    <div class="inline-discussions-nav"></div>
+    <div class="hd hd-3 subsection-title"><%- gettext('Specify whether discussion topics are divided') %></div>
+    <div class="division-scheme-container">
+        <div class="division-scheme-items">
+            <% for (var i = 0; i < availableSchemes.length; i++) { %>
+                <div class="division-scheme-item <% if (!availableSchemes[i].enabled) { %>is-hidden<% } %>">
+                    <label class="division-scheme-label">
+                        <input class="division-scheme" type="radio" name="division-scheme" id="<%- availableSchemes[i].key %>"
```

Comment:
We generally try to avoid IDs, and prefer to use classes (because IDs must be unique on the page). Was there any reason why an ID was necessary here?